pkg/policy/validate.go

```diff
@@ -255,51 +270,43 @@ func Validate(policy *kyverno.ClusterPolicy, client *dclient.Client, mock bool,
 		match := rule.MatchResources
 		exclude := rule.ExcludeResources
 		for _, value := range match.Any {
-			if !utils.ContainsString(value.ResourceDescription.Kinds, "*") {
-				err := validateKinds(value.ResourceDescription.Kinds, mock, client, *policy)
-				if err != nil {
-					return errors.Wrapf(err, "the kind defined in the any match resource is invalid")
-				}
```

Comment:
we need this changes, may be get removed while resolving conflict ?

Reply:
I have rebased and fixed this change.